gtsfm/two_view_estimator.py

```diff
@@ -159,7 +168,9 @@ def compute_correspondence_metrics(
 
 
 def compute_relative_pose_metrics(
-    i2Ri1_computed: Optional[Rot3], i2Ui1_computed: Optional[Unit3], i2Ti1_expected: Pose3,
```

Comment:
i dont think this violates the 120 width limit on the line.

Reply:
just updated it